pulsar-client-cpp/python/pulsar/functions/serde.py

```diff
@@ -75,6 +75,8 @@ def __init__(self):
   def serialize(self, input):
     if type(input) in self._types:
       return str(input).encode('utf-8')
+    if type(input) == bytes:
+      return input
     raise TypeError
```

Comment:
We should also add context on the type that we got passed here

Reply:
Added